src/ZoneServer/World/Actors/Characters/Character.cs

```diff
@@ -720,7 +722,16 @@ public void Heal(float hpAmount, float spAmount)
 			if (hpAmount == 0 && spAmount == 0)
 				return;
 
-			this.ModifyHpSafe(hpAmount, out var hp, out var priority);
+			float healingReduction = 0;
+
+			if (this.TryGetBuff(BuffId.DecreaseHeal_Debuff, out Buff decreaseHealDebuff))
+			{
+				healingReduction = decreaseHealDebuff.Vars.GetFloat("DecreaseHeal_Debuff.HealingReduction");
```

Comment:
By putting the intended reduction into a variable on the buff and using it here, you're making it necessary for anyone who needs to check for this debuff to know the inner workings of that particular buff handler, to get the string right, and then correctly use whatever value you put in there. That's a lot of potential points of failure, which could all be alleviated by providing a static method on the `DecreaseHeal_Debuff` handler that does the job of checking for the buff and returning the specific multiplier to use, with clear documentation of what to expect from the method.

Reply:
it cannot be static since the value relies on the skill level (NumberArg1).

Reply:
I'll show you what I meant during the merge^^